src/components/ApplicationForm/ApplicationFormContent.jsx

```diff
@@ -0,0 +1,172 @@
+      ...prevData,
+      [inputType]: value,
+    }))
+  }
+
+  const handleClearInput = (inputType) => {
+    setInputs((prevData) => ({...prevData, [inputType]: ''}))
+    setErrors((prevErrors) => prevErrors.filter((error) => error !== inputType))
+  }
+
+  return (
+    <div>
+      <h4 className={css.formHeader}>Formularz zgłoszeniowy</h4>
+      <form className={css.form} onSubmit={handleSubmit}>
+        <fieldset className={css.fieldset}>
+          <TextComponent
+            label="name"
+            placeholder="name"
+            value={inputs.name}
+            type="text"
+            errors={errors}
+            onChange={(e) => handleChange(e, 'name')}
+            onClear={() => handleClearInput('name')}
+          />
+          <TextComponent
+            label="email"
+            placeholder="email"
+            value={inputs.email}
+            type="email"
+            errors={errors}
+            onChange={(e) => handleChange(e, 'email')}
+            onClear={() => handleClearInput('email')}
+          />
+          <TextComponent
+            label="city"
+            placeholder="city"
+            value={inputs.city}
+            type="text"
+            errors={errors}
+            onChange={(e) => handleChange(e, 'city')}
+            onClear={() => handleClearInput('city')}
+          />
+          <SelectComponent
+            value={inputs.role}
+            onChange={(e) => handleChange(e, 'role')}
+            options={selectOptions}
+            error="role"
+            errors={errors}
+            condition={inputs.role === 'Inne'}
+          />
+          {inputs.role === 'Inne' && (
+            <TextComponent
+              label="customRole"
+              placeholder="customRolePlaceholder"
+              value={inputs.customRole}
+              type="text"
+              errors={errors}
+              onChange={(e) => handleChange(e, 'customRole')}
+              onClear={() => handleClearInput('customRole')}
+            />
+          )}
+        </fieldset>
+        <div className={css.bottomArea}>
+          <CheckboxComponent
+            value={inputs.certificate}
+            onChange={(e) => handleChange(e, 'certificate')}
+            error="checkbox"
+            errors={errors}
+            variant="certificate"
+          />
+          <CheckboxComponent
+            value={inputs.permission}
+            onChange={(e) => handleChange(e, 'permission')}
+            error="checkbox"
+            errors={errors}
+            variant="consent"
+          />
+          <p className={css.formText}>{t('applicationForm.menu')}</p>
+          <div className={css.menuWrapper}>
+            <RadioComponent
+              error="menu"
+              value={inputs.menu}
+              options={radioOptions}
+              onChange={(e) => handleChange(e, 'menu')}
+              errors={errors}
+            />
+          </div>
+        </div>
+        <Button
+          type="submit"
```

Comment:
W Figmie mamy inny kolor buttona. Chyba trzeba zaprojektować nowego.

Reply:
To chodziło o to, że jest zablokowany dopóki nie mamy wszystkich niezbędnych info. Uzgodniłem, że przycisk zostaje taki, ale mamy blokadę wysłania formularza